pkg/cmd/release/create/create.go

```diff
@@ -245,7 +245,7 @@ func createRun(opts *CreateOptions) error {
 					headRef = "HEAD"
 				}
 			}
-			if prevTag, err := detectPreviousTag(headRef); err == nil {
+			if prevTag, err := detectPreviousTag(headRef); err == nil && generatedNotes == nil {
```

Comment:
Could we avoid calling `detectPreviousTag` in the first place if its result won't get used?

Reply:
Yeah, no problem! I guess I was over-prioritizing trying to avoid a level of indentation :laughing: